code/modules/projectiles/guncrafting/magazines.dm

```diff
@@ -0,0 +1,29 @@
+/obj/item/projectile/bullet/guncrafting_pistol
+	damage = 14
+
+/obj/item/ammo_casing/guncrafting_pistol
+	desc = "A pistol bullet casing."
+	caliber = "NT Custom"
+	projectile_type = /obj/item/projectile/bullet/guncrafting_pistol
+
+/obj/item/ammo_box/magazine/guncrafting_pistol
+	name = "pistol magazine (NT Custom)"
+	icon_state = "45-8"
+	ammo_type = /obj/item/ammo_casing/guncrafting_pistol
+	caliber = "NT Custom"
+	max_ammo = 8
+
+/obj/item/projectile/bullet/guncrafting_ar
+	damage = 3
```

Comment:
>3 damage
>30 max ammo base

Am i missing something or does unloading a full mag of an assault rifle not even do enough damage to crit a person